luigi/cmdline.py

```diff
@@ -40,5 +41,14 @@ def luigid(argv=sys.argv[1:]):
             logging.basicConfig(level=logging.INFO, format=luigi.process.get_log_format(),
                                 filename=os.path.join(opts.logdir, "luigi-server.log"))
         else:
-            logging.basicConfig(level=logging.INFO, format=luigi.process.get_log_format())
+            config = luigi.configuration.get_config()
+            logging_conf = None
+            if not config.getboolean('core', 'no_configure_logging', False):
+                logging_conf = config.get('core', 'logging_conf_file', None)
+                if logging_conf is not None and not os.path.exists(logging_conf):
+                    logging_conf = None
+            if logging_conf is not None:
```

Comment:
This code block can be simplified I think. Why first `logging_conf = None` and then `if logging_conf is not None`. Just introduce the variable in the code path that you use it.

Also, can you please also add `print()` statements saying `"Loading logging conf from file"` or `"Defaulting to only logging to stdout. You're highly advised to setup logging"`? Something like that.